assets/src/ImageBlockExtension.js

```diff
@@ -92,25 +92,25 @@ const addExtraControls = function() {
               title={ __( 'Planet4 Image Options' ) }
               initialOpen={ true }
             >
-              <label>Caption alignment</label>
+              <label htmlFor>Caption alignment</label>
```

Comment:
It seems weird that `React` accepts `htmlFor` as this way.
What do you think?

Reply:
Yea, it was a nice hack to avoid the errors generated by ESLint.

Reply:
Maybe we can use paragraphs instead of labels if we have nothing to write in the `htmlFor`? 🤔

Reply:
That might require some styling fixes too though

Reply:
It turns out writing just `htmlFor` like this results in the following console warning:
```
Warning: Received `true` for a non-boolean attribute `htmlFor`.

If you want to write it to the DOM, pass a string instead: htmlFor="true" or htmlFor={value.toString()}.
```
so we should probably find another solution!